src/main/java/org/team1540/robot2018/OI.java

```diff
@@ -0,0 +1,160 @@
+package org.team1540.robot2018;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.buttons.Button;
+import edu.wpi.first.wpilibj.buttons.JoystickButton;
+import org.team1540.base.Utilities;
+import org.team1540.base.triggers.AxisButton;
+import org.team1540.base.triggers.DPadButton;
+import org.team1540.base.triggers.DPadButton.DPadAxis;
+
+/*
+ * Button Mapping
+ * X: 3
+ * Y: 4
+ * A: 1
+ * B: 2
+ * Back: 7
+ * Start: 8
+ * LB: 5
+ * RB: 6
+ * Right joystick center: 9
+ * Left joystick center: 10
+ *
+ * Axis Mapping:
+ * Left analog trigger: 2 (Only positive values)
+ * Right analog trigger: 3 (Only positive values)
+ *
+ * Left joystick:
+ * Up/Down: 1 (Up is negative)
+ * Left/Right: 0 (Left is negative)
+ *
+ * Right joystick:
+ * Up/Down: 5 (Up is negative)
+ * Left/Right: 4 (Left is negative)
+ */
+
+
+public class OI {
+
+  public static Joystick driver = new Joystick(0);
+  public static Joystick copilot = new Joystick(1);
+
+  public static final int X = 3;
```

Comment:
Not following naming conventions?

Reply:
Static final immutable fields use constant case per [Google Style].(https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names)

Reply:
What about `back` and `start` then?

Reply:
Fair point, will refactor those.